libs/sdk-ui-kit/src/Icon/icons/DragHandle.tsx

```diff
@@ -0,0 +1,26 @@
+// (C) 2021 GoodData Corporation
+import React from "react";
+import classNames from "classnames";
+
+import { IIconProps } from "../typings";
+
+/**
+ * @internal
+ */
+export const DragHandle: React.FC<IIconProps> = ({ className, color }) => {
+    return (
+        <svg
+            className={classNames(className, "sdk-ui-kit-icon")}
```

Comment:
What is this class for?

Reply:
It is for further CSS positioning etc. I didn't want to wrap it into the `<div>` just for that purpose and I suppose we might want to be able to select the icon with the CSS selector in the future anyway.

Reply:
(perhaps, we should alter all the existing components and add this property as well)

Reply:
We decided to omit `sdk-ui-kit-icon` class name for now.